.eslintrc.json

```diff
@@ -7,6 +7,7 @@
 		"curly": "error",
 		"camelcase": ["error", {"properties": "never"}],
         "no-multiple-empty-lines": ["error", { "max": 1, "maxEOF": 1 }],
+		"no-multi-spaces": "error",
```

Comment:
looks like `jsonlint` does not enforce one indent style

Reply:
Something one should look at when getting rid of the grunt dependencies...